### PR TITLE
updates people and alumni

### PR DESCRIPTION
**Pull request by samgdotson**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests.md) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: _data/alumni.yml [hunk 1]
```diff
@@ -89,6 +89,19 @@
       icon: twitter
       name: Twitter
 
+- name: Yi Nok Yeung (Nicholas)
+  pic: /img/people/yeungn.jpg
+  dates: 2021-present
```
Comment (abachma2) [inline _data/alumni.yml:94]: Should this still say present?

**Review by abachma2 — CHANGES_REQUESTED** [review]
Thanks for updating the website @samgdotson. I just have the one small change. I wonder (probably a bigger conversation) if there is a particular order for people in the alumni list? It looks like there is a general order of when they joined the group, but it isn't strictly followed.

File: _data/alumni.yml (hunk 1 above)
Comment (samgdotson) [reply _data/alumni.yml]: Nope, thx

**Review by abachma2 — APPROVED** [review]
Thanks for making the change. Approving and merging!

**Pull request merged**